content/derisking-government-tech/about.md

```diff
@@ -3,7 +3,33 @@ title: About
 permalink: /derisking-government-tech/about/
 layout: layouts/page
 tags: derisking-government-tech
+description: History and authors of the 18F De-risking Government Technology Guide 2.0.
 sidenav: false
 ---
 
-TK
+{% include 'derisking-government-tech/chapter_image.html' img_path: "assets/derisking-government-tech/img/icon.svg" %}
+
+The De-risking Government Technology Guide 2.0 consolidates content from the two parts of the original De-risking Guide: the [State Software Budgeting Handbook]({{ "https://guides.18f.gov/derisking/state-field-guide/" | url }}), released in August 2019, and the [Federal Agency Field Guide]({{ "https://guides.18f.gov/derisking/federal-field-guide/" | url }}), released in September 2020. In addition, the new section “[Working with a vendor development team]{{ "/derisking-government-tech/vendor-management/" | url }}” includes detailed guidance about vendor management. 
+
+These updates were made possible by the Office of Management and Budget’s [Facing a Financial Shock initiative]({{ "https://www.performance.gov/cx/life-experiences/facing-a-financial-shock/" | url }}). From May 2023 to June 2024, this funding enabled an 18F team to assess the feasibility of solutions to common challenges within state government IT acquisition, such as knowledge and skill gaps, focus on planning over results, and bureaucracy. After identifying vendor management to be a major area of concern among agency staff, the team took this opportunity to revise the De-risking Guide with relevant content.
+
+The current guide reflects the contributions of many current and former 18F staff:
+
+* Authors of the State Software Budgeting Handbook  
+  * Robin Carnahan, Randy Hart, and Waldo Jaquith  
```

Comment:
suggestion (non-blocking): Nest bullets 4 spaces in for better visual differentiation in raw markdown